pkg/monitor/backenddisruption/disruption_backend_sampler.go

```diff
@@ -140,6 +140,8 @@ func NewAPIServerBackend(clientConfig *rest.Config, disruptionBackendName, path
 		return nil, err
 	}
 
+	// Kube-apiserver allows 34s timeout for api requests. We want to match it here.
+	timeout := 34 * time.Second
```

Comment:
Old code wanted some percentage of 20s (old total timeout) with a comment indicating it was to catch diff failure modes? Should we do the same percentage of 34s total here?

Reply:
Ah excellent, thanks, that explains why I wasn't seeing that comment where the change was made. :)